# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: docshell/test/browser/browser_browsingContext-03.js
# Commit: d513683e1fdb
# Full Hash: d513683e1fdbfdb79f5112e9745f2fc0fb4e92fe
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2019-08-15 09:46:49
# Regressor Bug: 1562292
# File Overlap Count: 1
# Description:
#   Bug 1562292: Part 1a - Consider requesting context in BrowsingContext lookup methods. r=farre
#   
#   Access to a particular named browsing context depends on the caller who is
#   attempting the access. For a call to `parent.open(..., name)`, for instance,
#   it's the privileges of the sub-frame making the open() call that matter, even
# ==============================================================================

diff -r 99d4d3837b3b -r d513683e1fdb docshell/test/browser/browser_browsingContext-03.js
--- a/docshell/test/browser/browser_browsingContext-03.js	Wed Aug 14 19:37:05 2019 +0300
+++ b/docshell/test/browser/browser_browsingContext-03.js	Tue Jul 30 17:30:55 2019 -0700
@@ -138,7 +138,10 @@
                 null,
                 false
               );
-              let browsingContext = browsingContexts[i].findWithName("target");
+              let browsingContext = browsingContexts[i].findWithName(
+                "target",
+                browsingContexts[i]
+              );
               is(
                 docShell ? docShell.browsingContext : null,
                 browsingContext,
@@ -155,7 +158,10 @@
                 null,
                 false
               );
-              let browsingContext = browsingContexts[i].findWithName(target);
+              let browsingContext = browsingContexts[i].findWithName(
+                target,
+                browsingContexts[i]
+              );
               is(
                 docShell ? docShell.browsingContext : null,
                 browsingContext,